package.json

```diff
@@ -35,7 +35,8 @@
     "remark-html": "13.0.1",
     "remark-preset-lint-recommended": "5.0.0",
     "slugify": "1.6.5",
-    "styled-jsx": "3.3.0"
+    "styled-jsx": "3.3.0",
+    "yarn": "^1.22.19"
```

Comment:
I don't think having `yarn` as a dependency will change the version that is used to build the site. There's information in this section of the readme about updating the version:
https://github.com/neontribe/www#wrench-maintenance-construction